README.md

```diff
@@ -127,6 +128,9 @@ At the time of writing there is a [proposal](https://github.com/privacycg/storag
 ## Non-goals
 
 - This document does not describe any changes to how a top-level site interacts with its own cookies.
+  For top-level site owners, most partitioned cookie use cases are covered by using SameSite=Lax/Strict instead.
+  However, there are edge cases where CHIPS may be useful to top-level site owners.
```

Comment:
"uncommon cases" may be more appropriate than "edge cases"?

Reply:
Done.